apps/server/src/modules/files-storage/files-storage.const.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilesStorageInternalActions {
```

Comment:
Maybe we should call them "paths" instead of actions? (because they are paths and are used as such)